site/index.html

```diff
@@ -76,6 +76,52 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and help you growing.
+               The strong type system makes it easy to write programs that work, both large and small, even when you don't like writing types &mdash; the compiler will figure it out.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Scale with ease</h3>
+            <p>From 1 to 100k users, and more. Haskell offers one of the best support for async, concurrent and parallel programming.
+               It can take advantage of multi-core processors with ease and confidence like few other languages can
+               &mdash; while keeping complexity low as your user base and code base grows.
```

Comment:
I feel there are some repetitive words, especially "confidence". Additionally, I think we should mention why concurrency works so well:

```suggestion
            <p>From 1 to 100k users, and more. Haskell offers one of the best support for async, concurrent and parallel programming like few other languages can.
               It can take advantage of multi-core processors with ease thanks to green threads and garbage collection. Additionally, you can use advanced streaming libraries for efficient data processing
               &mdash; all while keeping complexity low as your user base and code base grow.
```